pkg/state/state_skel.go

```diff
@@ -34,6 +34,13 @@ import (
 	"github.com/Mellanox/network-operator/pkg/render"
 )
 
+type nodeInfoRuntimeSpec struct {
```

Comment:
how is this change related to the cleanup?

Reply:
same above

Reply:
same as above

Reply:
updated